diff --git a/scripts/build-ffmpeg.py b/scripts/build-ffmpeg.py
index 1788e6b..2e272d7 100644
--- a/scripts/build-ffmpeg.py
+++ b/scripts/build-ffmpeg.py
@@ -55,7 +55,7 @@ if not os.path.exists(output_tarball):
             run(["where", tool])
 
     with log_group("install python packages"):
-        run(["pip", "install", "cmake", "meson", "ninja"])
+        run(["pip", "install", "meson"])
 
     # build tools
 
