diff --git a/scripts/cibuildpkg.py b/scripts/cibuildpkg.py
index e7b941c..ca420c6 100644
--- a/scripts/cibuildpkg.py
+++ b/scripts/cibuildpkg.py
@@ -85,7 +85,7 @@ def make_args(*, parallel: bool) -> List[str]:
     args = []
 
     # do not parallelize build when running in qemu
-    if parallel and platform.machine() not in ["aarch64", "ppc64le", "s390x"]:
+    if parallel and platform.machine() not in ["aarch64", "ppc64le", "s390x", "armv7l"]:
         args.append("-j")
 
     return args
