diff --git a/scripts/build-ffmpeg.py b/scripts/build-ffmpeg.py
index 1788e6b..023d3ec 100644
--- a/scripts/build-ffmpeg.py
+++ b/scripts/build-ffmpeg.py
@@ -97,7 +97,7 @@ if not os.path.exists(output_tarball):
         ),
         Package(
             name="gmp",
-            source_url="https://gmplib.org/download/gmp/gmp-6.2.1.tar.xz",
+            source_url="https://github.com/bjia56/armv7l-wheels/releases/download/build-dependency-archives/gmp-6.2.1.tar.xz",
             # out-of-tree builds fail on Windows
             build_dir=".",
         ),
